src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -681,14 +682,47 @@ public AlignedAssemblyOrExcuse apply( final Tuple2<Integer, List<SVFastqUtils.Fa
 
         final List<SVInterval> intervals = new ArrayList<>(allEvidence.size());
         while ( evidenceIterator.hasNext() ) {
-            intervals.add(evidenceIterator.next().getLocation());
+            //intervals.add(evidenceIterator.next().getLocation());
+            intervals.addAll( splitInterval( evidenceIterator.next().getLocation(), params ) );
         }
 
         evidenceRDD.unpersist();
 
         return intervals;
     }
 
+    /** Split SVInterval into length-numIntervals List<SVInterval> with requested overlap **/
+    public static List<SVInterval> splitInterval( final SVInterval interval, final Params params ) {
+        List<SVInterval> splitIntervals;
+        final int originalLen = interval.getLength();
+        if( originalLen <= params.maxIntervalLength ) {
+            splitIntervals = new java.util.ArrayList<SVInterval>();
+            splitIntervals.add(interval);
+        }
+        else {
+            // Split interval into numIntervals SVIntervals. Nominally each has length exactly equal to maxLen, and
+            // overlap at least equal to requested amount. Matching condition, up to rounding error:
+            //     originalLen ~ maxLen * numIntervals - overlapLen * (numIntervals - 1)
+            final int numIntervals = (int)Math.ceil( (originalLen - params.splitIntervalMinOverlap)
+                    / (double)(params.maxIntervalLength - params.splitIntervalMinOverlap ) );
+            // If the interval is *slightly* larger than params.maxIntervalLength, you may end up with an absurd amount
+            // of overlap. In this case, shorten the split intervals a bit
+            final int maxOverlap = Math.max( params.maxIntervalLength / 3, params.splitIntervalMinOverlap );
+            final int overlap = (numIntervals * params.maxIntervalLength - originalLen) / (numIntervals - 1);
+            final int splitIntervalLen = (numIntervals == 2 && overlap > maxOverlap) ?
+                    (originalLen + maxOverlap) / 2:
+                    params.maxIntervalLength;
+            // Have numIntervals and splitIntervalLen, so make the intervals
+            final double startShift = (originalLen - splitIntervalLen) / (double)(numIntervals - 1);
+            splitIntervals =
+                    java.util.stream.IntStream.range(0, numIntervals)
+                            .map( (idx) -> interval.getStart() + (int)Math.round( startShift * (double)idx ) )
+                            .mapToObj( (startIdx) -> new SVInterval( interval.getContig(), startIdx, startIdx + splitIntervalLen ) )
+                            .collect( java.util.stream.Collectors.toList() );
```

Comment:
It totally doesn't matter in this case (total number of intervals to check is small and numIntervals is generally quite modest), but since you're just learning idiomatic Java I'll point this out:
Collectors.toList() doesn't know how many elements you're going to end up with, so it can't be very efficient.  There's no guarantee on these details, but what our JVM does is to allocate an ArrayList of default size, which is 10, and then grows it by 50% every time it runs out of room.  That's a lot of reallocation and copying if the list is large.
But you know the size of the resulting list ahead of time.  So do this instead:
```
.collect( SVUtils.arrayListCollector(numIntervals) );
```
That will allocate an ArrayList with an adequate initial capacity, and Bob's your uncle.
Even when you don't know the exact size of the final list, it's good to take a guess so that you don't end up reallocating a tiny list a gazillion times.